### PR TITLE
release27-2-24

### PR DESCRIPTION
**Pull request by PK106552**
### Summary

**What was changed:**

The following packages will be released:
 
  - terra-abstract-modal@3.47.0
  - terra-application-layout@5.83.0
  - terra-application-links@6.75.0
  - terra-application-navigation@1.77.0
  - terra-collapsible-menu-view@6.90.0
  - terra-date-input@1.53.0
  - terra-date-picker@4.100.0
  - terra-date-time-picker@4.108.0
  - terra-dialog-modal@3.104.0
  - terra-file-path@1.7.0
  - terra-folder-tree@1.0.0
  - terra-form-validation@1.100.0
  - @cerner/terra-framework-docs@1.68.0
  - terra-menu@6.85.0
  - terra-modal-manager@6.72.0
  - terra-navigation-prompt@1.84.0
  - terra-notification-dialog@4.41.0
  - @cerner/terra-pills@1.19.0
  - terra-popup@6.75.0
  - terra-slide-panel-manager@5.91.0
  - terra-tabs@7.18.0
  - terra-time-input@4.64.0
**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-folder-tree/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.0.0 - (February 27, 2024)
```
Comment (saket2403) [inline packages/terra-folder-tree/CHANGELOG.md:5]: This should be alpha9

Comment (PK106552) [reply packages/terra-folder-tree/CHANGELOG.md]: Updated

Comment (PK106552) [reply packages/terra-folder-tree/CHANGELOG.md]: Updated

File: packages/terra-folder-tree/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "terra-folder-tree",
-  "version": "1.0.0-alpha.8",
+  "version": "1.0.0",
```
Comment (sycombs) [inline packages/terra-folder-tree/package.json:3]: This version does not match the changelog version. It should be 1.0.0-alpha.9.

**Comment by PK106552** [comment]
New release PR has been created with latest merge - https://github.com/cerner/terra-framework/pull/2054

**Pull request closed without merging**